frontend/Auth/ResetPassword.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import { Button, ButtonKind } from "../Styleguide/Button";
+import { FormField } from "../Styleguide/FormField";
+import { FormFieldLabel } from "../Styleguide/FormFieldLabel";
+import { Input } from "../Styleguide/Input";
+import { RouterProps } from "react-router";
+import { useMutation } from "react-query";
+import { flaxFetch } from "../Utils/flaxFetch";
+import { unary } from "lodash-es";
+
+export function ResetPassword(props: RouterProps) {
+  const [resetPasswordFormData, setResetPasswordFormData] =
+    useState<ResetPasswordFormData>({
+      email: "",
+    });
+
+  const history = useHistory();
+
+  const submitMutation = useMutation<
+    ResetPasswordResultData,
+    Error,
+    React.FormEvent<HTMLFormElement>
+  >(
+    (evt) => {
+      evt.preventDefault();
+      let requestPromise = flaxFetch<ResetPasswordResultData>(
+        `/send-reset-password-email`,
+        {
+          method: "POST",
+          body: {
+            email: resetPasswordFormData.email,
+          },
+        }
+      );
+      return requestPromise;
+    },
+    {
+      onSuccess: async (data, variables, context) => {
+        history.push({
+          pathname: "/reset-password-email-sent",
+          state: { email: resetPasswordFormData.email },
+        });
+      },
+      onError: (error, variables, context) => {
+        console.log(error);
```

Comment:
Console log isn't the best way to record errors - they ideally should be logged to a server so we can see and diagnose them. Sentry and Bugsnag are two popular projects that accomplish this. We don't have them in place right now, but how they work is they require errors to be thrown in order to report them to the server. So we should change this to throw error rather than console log